### PR TITLE
sg: pass parameters from bootstrap.sh

### PR DESCRIPTION
**Pull request by abitrolly**
To help with unattended installs, like in Gitpod.

## Test plan

Ensure that parameters are passed to `sg install` when running bootstrap script from terminal.

    ./bootstrap.sh -f

Ensure that parameters are passed when piping the script to shell.

    cat ./bootstrap.sh | sh -s -- -f -p

### CODE REVIEW

**Comment by keegancsmith** [comment]
Kicked off CI for ya https://buildkite.com/sourcegraph/sourcegraph/builds/180510

**Comment by abitrolly** [comment]
@keegancsmith thanks!

**buildkite/sourcegraph** job had failed, but I can't see why, because..

image

**Review by burmudar — APPROVED** [review]

**Comment by burmudar** [comment]
> @keegancsmith thanks!
> 
> **buildkite/sourcegraph** job had failed, but I can't see why, because..
> 
> image

Small linting failure. I'll fix it and kick off another run!
Screenshot 2022-10-31 at 07 39 12

File: dev/sg/bootstrap.sh [hunk 1]
```diff
@@ -58,8 +58,8 @@ main() {
     exit 1
   fi
 
-  printf '%s\n' 'running sg install' 1>&2
-  "$_file" install </dev/tty
+  printf 'running "%s %s"\n' 'sg install' "$*" 1>&2
+  "$_file" install $* </dev/tty
```
Comment (burmudar) [inline dev/sg/bootstrap.sh:62]:
```suggestion
  "$_file" install "$*" </dev/tty
```

**Review by mrnugget — APPROVED** [review]

**Comment by burmudar** [comment]
Thanks @abitrolly!

**Comment by abitrolly** [comment]
@burmudar I just tested it and quoting in the last commit breaks the script if two parameters are specified.

So instead of executing `sg install -f -p` with two argument, it executes it with just one quoted argument `sg install "-f -p"` resulting in the error.
```
Incorrect Usage: flag provided but not defined: -f -p
```

**Comment by burmudar** [comment]
Ah damn. Thanks for picking it up. I'll disable the linter for that line

**Comment by burmudar** [comment]
@abitrolly created a fix here https://github.com/sourcegraph/sourcegraph/pull/43682

**Pull request merged**